backbone/src/main/java/org/researchstack/backbone/answerformat/BirthDateAnswerFormat.java

```diff
@@ -0,0 +1,50 @@
+package org.researchstack.backbone.answerformat;
+import org.researchstack.backbone.R;
+import org.researchstack.backbone.ui.step.body.BodyAnswer;
+
+import java.util.Calendar;
+import java.util.Date;
+
+public class BirthDateAnswerFormat extends DateAnswerFormat
+{
+    private final int minAge;
+    private final int maxAge;
+
+    private static Date dateFromAge(int age)
+    {
+        Calendar calendar = Calendar.getInstance();
+        if(age != 0)
+        {
+            calendar.add(Calendar.YEAR, - age);
+            return calendar.getTime();
+        }
+
+        return null;
+    }
+
+    public BirthDateAnswerFormat(Date defaultDate, int minAge, int maxAge)
+    {
+        super(DateAnswerStyle.Date, defaultDate, dateFromAge(maxAge), dateFromAge(minAge));
+        this.minAge = minAge;
+        this.maxAge = maxAge;
+    }
+
+    @Override
+    public BodyAnswer validateAnswer(Date resultDate)
+    {
+        Date minDate = getMinimumDate();
+        Date maxDate = getMaximumDate();
+
+        if(minDate != null && resultDate.getTime() > minDate.getTime())
```

Comment:
HH mm ss may mess this up if the resultDate lands on the same day (mm dd yy) as min or max.

Either needs to Set min, max, result w/ a time of 00:00:00 OR only compare year month then day.